src/Player.ts

```diff
@@ -0,0 +1,46 @@
+import { Herd } from './Herd';
+
+class Player {
+  protected name: string;
+  protected avatar: string;
+  protected herd: Herd;
+  protected score: number;
+
+  // TODO: set path to default avatar when it's available
+  constructor(name: string, avatar = 'path to default avatar') {
+    this.name = name;
+    this.herd = new Herd();
+    this.avatar = avatar;
+    this.score = 0;
+  }
+
+  get theName() {
+    return this.name;
+  }
+
+  get theAvatar() {
+    return this.avatar;
+  }
```

Comment:
Types are missing.
For getters, I belive it should look like this:
```.ts
 get theName(): string {
    return this.name;
  }

Reply:
Getters and setters doesn't need type. TS is clever, as he knows type of name (in this case), he knows also returning type of getter.